### PR TITLE
Move template and logo for multi-sample report to Docker

### PR DESCRIPTION
**Pull request by JureZmrzlikar**

### CODE REVIEW

File: resolwe_bio/docker_images/latex/Dockerfile [hunk 1]
```diff
@@ -4,3 +4,7 @@ MAINTAINER Resolwe Bioinformatics authors https://github.com/genialis/resolwe-bi
 
 # Force matplotlib backend to a headless one.
 ENV MPLBACKEND Agg
+
+# Additional files:
+COPY files/multireport-template.tex /opt/resolwebio/multireport-template.tex
```
Comment (mstajdohar) [inline resolwe_bio/docker_images/latex/Dockerfile]: Please choose a more explicit name. This is a general `Latex` docker image, we may want to have more templates and we want to know which template is which.

**Comment by jkokosar** [comment]
Please rebase

**Comment by JureZmrzlikar** [comment]
Rebased.

File: resolwe_bio/docker_images/latex/Dockerfile [hunk 2]
```diff
@@ -4,3 +4,7 @@ MAINTAINER Resolwe Bioinformatics authors https://github.com/genialis/resolwe-bi
 
 # Force matplotlib backend to a headless one.
 ENV MPLBACKEND Agg
+
+# Additional files:
+COPY files/template_amplicon_report_multi_sample.tex /opt/resolwebio/template_amplicon_report_multi_sample.tex
```
Comment (kostko) [inline resolwe_bio/docker_images/latex/Dockerfile]: Please collapse the number of file copy operations to reduce the number of layers (e.g. copy the whole directory into `/assets` or something). I would also rename `files` to `assets`.

Comment (JureZmrzlikar) [reply resolwe_bio/docker_images/latex/Dockerfile]: Good point, fixed.

**Pull request merged**